packages/main/src/ColorPalette.ts

```diff
@@ -168,7 +168,8 @@ class ColorPalette extends UI5Element {
 	_itemNavigationRecentColors: ItemNavigation;
 	_recentColors: Array<string>;
 	moreColorsFeature?: ColorPaletteMoreColors;
-	_currentlySelected?: ColorPaletteItem;
+	_currentlySelected?: ColorPaletteItem | null;
```

Comment:
Remove `?` and intiialze the member inside the constructor with default value null

this._currentlySelected = null;